tests/sentry/demo/test_demo_org_manager.py

```diff
@@ -208,3 +212,37 @@ def test_create_mobile_demo_org(
         assert len(Project.objects.filter(organization=org)) == 5
         mock_handle_scenario.assert_called_once_with(mock.ANY, mock.ANY)
         mock_handle_mobile_scenario.assert_called_once_with(mock.ANY, mock.ANY, mock.ANY)
+
+    @override_settings(DEMO_DATA_QUICK_GEN_PARAMS=DEMO_DATA_QUICK_GEN_PARAMS)
+    @mock.patch.object(DataPopulation, "handle_react_python_scenario")
+    @mock.patch.object(DataPopulation, "handle_mobile_scenario")
+    @mock.patch("sentry.demo.demo_org_manager.generate_random_name", return_value=org_name)
+    @mock.patch.object(DataPopulation, "generate_releases")
+    def test_get_config_var(
```

Comment:
@roggenkemper I think this belongs in a `test_data_population.py` file where we can omit a lot of the setup that's needed for a demo org. The org can really just be a normal org too: `DataPopulation` doesn't care what kind of org it is.

Reply:
makes sense!